### DISCUSSION TITLE
Blank page numbered

### DISCUSSION THREAD

**Discussion by roxooo** [joaomlourenco/novathesis#303] [Q&A] [answered]
currently using the version 6.9.4 of the template and I commented all the glossaries except for the acronyms and there seems to be a blank page which is numbered between the list of tables and the acronyms. I guess it is meant for one of the glossaries but again, the only uncommented is the acronyms section.

thanks in advance!
![WhatsApp Image 2022-02-14 at 18 04 09](https://user-images.githubusercontent.com/91677308/153931778-2770c2d5-c3c7-48d7-8b62-1c3fe013cad8.png)

**Comment by joaomlourenco** [comment]
@roxooo I believe that is a bug.  I have to check.  To make my like easier analysing the problem, may I ask you to allow me to access you document in Overleaf?  Please share it with my **FCT email** "joao.lourenco@fct...etc...".

**Reply by roxooo** [reply]
thank you very much! I just sent the request

**Reply by joaomlourenco** [reply]
@roxooo  Received.  Already did a first check. It is unclear for me what is the source of the problem. Needs further investigation.  I'll let you know when I have some news.

**Comment by joaomlourenco** [comment, marked as answer]
Fixed in v6.9.6.

**Reply by RenataMartins** [reply]
I'm using version 7.0.2 and I have the same problem... Could you please help me?

**Comment by joaomlourenco** [comment]
I cannot reproduce the bug.
Please try with the latest version (7.0.3).
